artifacthub-repo.yml

```diff
@@ -0,0 +1,8 @@
+# Artifact Hub repository metadata file
+owners: # (optional, used to claim repository ownership)
+  - name: Vincent Demeester
+    email: vincent@sbr.pm
```

Comment:
@vdemeester probably you can add me as well 😉

Reply:
@vinamra28 I want to add everyone in owners, but I need the email from your profile in artifacthub. I would suggest, we get this in the repo, we claim owner ship, and we can open PRs to add ourslevs, make sense ?

Reply:
definitely, makes sense